resources/js/views/gallery-panels/Photo.vue

```diff
@@ -0,0 +1,345 @@
+				<img
+					v-if="imageViewMode == 2"
+					id="image"
+					alt="medium"
+					class="absolute m-auto w-auto h-auto animate-zoomIn bg-contain bg-center bg-no-repeat"
+					:src="photo.size_variants.medium?.url ?? ''"
+					:class="is_full_screen || is_slideshow_active ? 'max-w-full max-h-full' : 'max-wh-full-56'"
+					:srcset="srcSetMedium"
+				/>
+				<img
+					v-if="imageViewMode == 3"
+					id="image"
+					alt="big"
+					class="absolute m-auto w-auto h-auto animate-zoomIn bg-contain bg-center bg-no-repeat"
+					:class="is_full_screen || is_slideshow_active ? 'max-w-full max-h-full' : 'max-wh-full-56'"
+					:style="style"
+					:src="photo?.size_variants.original?.url ?? ''"
+				/>
+				<!-- This is a livephoto : medium -->
+				<div
+					v-if="imageViewMode == 4"
+					id="livephoto"
+					data-live-photo
+					data-proactively-loads-video="true"
+					:data-photo-src="photo?.size_variants.medium?.url"
+					:data-video-src="photo?.live_photo_url"
+					class="absolute m-auto w-auto h-auto"
+					:class="is_full_screen || is_slideshow_active ? 'max-w-full max-h-full' : 'max-wh-full-56'"
+					:style="style"
+				></div>
+				<!-- This is a livephoto : full -->
+				<div
+					v-if="imageViewMode == 5"
+					id="livephoto"
+					data-live-photo
+					data-proactively-loads-video="true"
+					:data-photo-src="photo?.size_variants.original?.url"
+					:data-video-src="photo?.live_photo_url"
+					class="absolute m-auto w-auto h-auto"
+					:class="is_full_screen || is_slideshow_active ? 'max-w-full max-h-full' : 'max-wh-full-56'"
+					:style="style"
+				></div>
+
+				<!-- <x-gallery.photo.overlay /> -->
+			</div>
+			<NextPrevious
+				v-if="photo.previous_photo_id !== null"
+				:albumId="props.albumid"
+				:photoId="photo.previous_photo_id"
+				:is_next="false"
+				:style="previousStyle"
+			/>
+			<NextPrevious
+				v-if="photo.next_photo_id !== null"
+				:albumId="props.albumid"
+				:photoId="photo.next_photo_id"
+				:is_next="true"
+				:style="nextStyle"
+			/>
+			<Overlay :photo="photo" :image-overlay-type="lycheeStore.image_overlay_type" />
+			<div
+				v-if="photo?.rights.can_edit && !is_edit_open"
+				class="absolute top-0 sm:h-1/4 w-full sm:w-1/2 left-1/2 -translate-x-1/2 opacity-50 lg:opacity-10 group lg:hover:opacity-100 transition-opacity duration-500 ease-in-out z-20 mt-14 sm:mt-0"
+				:class="is_slideshow_active ? 'hidden' : ''"
+			>
+				<span class="absolute left-1/2 -translate-x-1/2 p-1 min-w-[25%] w-full filter-shadow text-center">
+					<DockButton
+						icon="star"
+						:class="photo.is_starred ? 'fill-yellow-500 lg:hover:fill-yellow-100' : 'fill-white lg:hover:fill-yellow-500'"
+						v-tooltip.bottom="photo.is_starred ? $t('lychee.UNSTAR') : $t('lychee.STAR')"
+						v-on:click="toggleStar()"
+					/>
+					<DockButton
+						pi="image"
+						:class="photo.is_starred ? 'fill-yellow-500 lg:hover:fill-yellow-100' : 'fill-white lg:hover:fill-yellow-500'"
```

Comment:
It's using starred instead of header, but it doesn't seem to be yellow for me (or highlighting at all). That's based on a293683 so it may just be me. It's also not a big deal, but all of the other buttons change colour on mouseover :P

Reply:
I checked this does nothing indeed. Fixed [a47bcc9](https://github.com/LycheeOrg/Lychee/pull/2504/commits/a47bcc997ea8a82dcf337e1c25946a403858903b).
I am not able to get the header info, so it is just a button that goes blue when hovered.